wp-content/themes/core/components/card/css/card.pcss

```diff
@@ -100,6 +118,17 @@
 			margin-bottom: 0;
 		}
 	}
+
+	/* CASE: Style: Search */
+	.c-card--style-search & {
+		display: none;
+
+		@media (--viewport-medium) {
+			display: block;
+			width: 168px;
```

Comment:
I think you can use: `--grid-2-col` for this value (so it can automatically adjust if the grid size changes).